File: Task-03/C++/Sub-Task-2.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream inputFile("input.txt");
    std::ofstream outputFile("output.txt");

    std::string line;
    while (getline(inputFile, line)) {
        outputFile << line << std::endl;
    }
    inputFile.close();
    outputFile.close();

    return 0;
}

File: Task-03/Rust/Sub-Task-2.rs
use std::fs::File;
use std::io::{self, Read, Write};

fn main() -> io::Result<()> {
    let mut inputfile = File::open("input.txt")?; 
    let mut contents = String::new();
    input_file.read_to_string(&mut contents)?; 

    let mut output_file = File::create("output.txt")?; 
    output_file.write_all(contents.as_bytes())?; 
    Ok(())
}
